meson.build

```diff
@@ -269,6 +270,9 @@ endif
 if cc.has_function('realpath')
   conf.set('HAVE_REALPATH', '1')
 endif
+if cc.has_function('sigaction')
```

Comment:
Do we have any idea if current versions of clang and gcc both support this going in?

Reply:
Well, sigaction is a pretty old interface -- I'd be amazed if all libc's didn't support it.